### PR TITLE
Upgrade kafka version 1.1.1

### PR DESCRIPTION
**Pull request by eskabetxe**
upgrade kafka version to the most recent posible

### CODE REVIEW

**Comment by eskabetxe** [comment]
@highker could you check this

File: presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java [hunk 1]
```diff
@@ -121,9 +123,10 @@ public void createTopics(int partitions, int replication, Properties topicProper
         checkState(started.get() && !stopped.get(), "not started!");
 
         ZkClient zkClient = new ZkClient(getZookeeperConnectString(), 30_000, 30_000, ZKStringSerializer$.MODULE$);
+        ZkUtils zkUtils = ZkUtils.apply(zkClient, false);
```
Comment (wenleix) [inline presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java]: Curious: Looks like this variable is only used once. Can we inline it ?

Comment (eskabetxe) [reply presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java]: done

**Comment by eskabetxe** [comment]
@highker done

**Pull request merged**